addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -76,9 +76,18 @@
             &:not(.o_handle_cell):not(.o_list_button):not(.o_list_record_remove) {
                 --fieldWidget-display: block; // Will affect inner components (see fields.scss)
 
+                // FIXME: ugly temporary overrides to visually align the field widget
+                // icons in the list view without the vertical-align: middle;
+                --fieldTaskStateSelection-margin: .2em;
+                --fieldMany2oneAvatarResource-margin: .25em;
```

Comment:
Sorry to insist but... why absolutely wanting to set those values in `web` ? :thinking:

Reply:
Because they should only get this margin if they are inside a list_view, setting the variable here assure that it will get applied in the list_view only 😄